docs(faq): add why react-native-firebase (#5950)
invertase/react-native-firebase: docs/faqs-and-tips.md

# FAQs

### Why `react-native-firebase` over `firebase-js-sdk`?

This package wraps `firebase-android-sdk` and `firebase-ios-sdk` into a Javascript API for React Native projects, so the main benefits come with the access to native code.

> - There are more modules in the native SDKs than the web SDK because some things only make sense in a mobile / native context (Dynamic Links, App Distribution, Crashlytics), so you can actually do more, and some of it is important for e.g., app virality, like Dynamic Links or quality (crash reporting)

Reviewer: ```suggestion
- There are more modules in the native SDKs than the web SDK because some things only make sense in a mobile / native context (Dynamic Links, App Distribution, Crashlytics), so you can actually do more, and some of it is important for example, making an app more viral with Dynamic Links or monitoring quality with Crashlytics
```
